llm/ext_server/server.cpp

```diff
@@ -2527,6 +2528,10 @@ static void server_params_parse(int argc, char **argv, server_params &sparams,
         {
             params.cont_batching = true;
         }
+        else if (arg == "-fa" || arg == "--flash-attn")
```

Comment:
maybe write the if in the same style as 2511

} else if condition {
}

Reply:
I can change it if you like, however my reasons for how it is right now:

- The style of line 2511 seems less common as most of the other statements are formatted like I've got it here.
- I kept the style the same as the statements directly above and below.
- My IDE's (VSCode) default CPP linter didn't like that and wanted to change it back to how I've got it here.

Either way - happy to change it if required, just let me know.

Reply:
I think consistency is the key for readability and maintenance in the long run. 
To keep the consistency, maybe minimize the change by updating the line 2511 instead.

Reply:
Yeah fair enough, I've aligned the outliers now.